definitions/pipelines/somatic_exome.cwl

```diff
@@ -331,7 +341,12 @@ outputs:
         type: File?
         outputSource: manta/tumor_only_variants
         secondaryFiles: [.tbi]
-
+    somalier_pairs:
```

Comment:
I'd suggest labelling these two outputs something more descriptive that describes what they contain: "concordance_metrics_something..."?